### PR TITLE
🐛 Fallback on invalid story animation attributes

### LINKED ISSUE #29184: Story page navigation fails with wrong animation presets values
Story page navigation fails with wrong animation presets values, cf https://www.usatoday.com/web-stories/7-things-you-can-do-to-increase-your-emotional-intelligence/ that has `animate-in-delay="nans"`

We should:
1. Keep logging errors to help publishers
2. If the provided attribute has an error, fallback to the default value (eg: `0` for `animation-delay`). `animate-in-delay` or `scale-start` or `translate-x` all fail page navigations.
3. Alternatively if (2) is too hard to implement or doesn't give a satisfying result, we should display the element without playing its animation. No matter if we end up doing (2) or (3), we do want the content to be visible to users.

cc @ampproject/wg-stories @alanorozco

### PR DESCRIPTION
**Pull request by alanorozco**
Fixes #29184

### CODE REVIEW

File: extensions/amp-story/1.0/test/test-animation.js [hunk 1]
```diff
@@ -542,7 +542,9 @@ describes.realWin('amp-story animations', {}, (env) => {
         `,
         ampdoc
       );
-      return expect(() => animationManager.applyFirstFrame()).to.throw();
+      return allowConsoleError(() =>
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/test/test-animation.js:545]: Double checking: it throws but doesn't prevent page navigation like on the example story?

Comment (alanorozco) [reply extensions/amp-story/1.0/test/test-animation.js:545]: This is an unrelated test, this is for the preset name (`animate-in=`) which is handled the same as before. (Test started to complain without `allowConsoleError` for some reason).

Would we like to just warn when the preset name is invalid?

File: extensions/amp-story/1.0/utils.js [hunk 2]
```diff
@@ -33,22 +33,19 @@ import {user, userAssert} from '../../../src/log';
...
-  const units = match[2];
+  const num = match ? match[1] : undefined;
+  const units = match ? match[2] : undefined;
 
-  userAssert(
-    match && match.length == 3 && (units == 's' || units == 'ms'),
-    'Invalid time string %s',
-    time
-  );
+  if (!match || match.length != 3 || (units != 's' && units != 'ms')) {
```
Comment (Enriqe) [inline extensions/amp-story/1.0/utils.js:45]:
```suggestion
  if (!match || match.length !== 3 || (units !== 's' && units !== 'ms')) {
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # AMP ⚡
 
+⚡⚡⚡
+
+Do you build things with AMP? If you do, please fill out the new [AMP Developer Survey](https://go.amp.dev/devx-survey)! We run this survey quarterly, so please fill this out even if you've responded to earlier surveys!
```
Comment (gmajoulet) [inline README.md:5]: Oops

Comment (alanorozco) [reply README.md:5]: I'm very confused, this is also part of this file on `master`.

Comment (alanorozco) [reply README.md]: (Rebased, fixed!)

**Pull request merged**